zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -964,8 +964,59 @@ property, when available, is noted below.
 * *digest.enabled* :
     (Java system property only: **zookeeper.digest.enabled**)
     **New in 3.6.0:**
-    The digest feature is added to self-verify the correctness inside
-    ZooKeeper when loading database from disk, and syncing with leader.
+    The digest feature is added to detect the data inconsistency inside
+    ZooKeeper when loading database from disk, catching up and following
+    leader, its doing incrementally hash check for the DataTree based on 
+    the adHash paper mentioned in mentioned in 
+
```

Comment:
- `mentioned in mentioned in`？
- Great doc, I got the idea.

Reply:
good catch on the doc error. +1